### PR TITLE
Sepolia resolvers

### PR DESCRIPTION
**Pull request by makoto**
## Change list

- Remove L2 deployment and moved to https://github.com/ensdomains/ens-contracts/pull/309
- Add L1 deployment details.
- Set reverse registrar namespace to `{cointype}.reverse.evmgateway.eth`
- Change `L2ReverseRegistrar` storage slots for verification

## Notes

- Arbitrum L1 resolver needs to be redeployed once subgraph on base is fully supported

### CODE REVIEW

File: arb-gateway/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ wrngler login
 wrangler secret put L1_PROVIDER_URL
 wrangler secret put L2_PROVIDER_URL
 wrangler secret put L2_ROLLUP
-yarn deploy
+yarn deploy --env sepolia
```
Comment (Arachnid) [inline arb-gateway/README.md:30]: Didn't you decide to use environment variables for this?

File: crosschain-resolver/README.md [hunk 2]
```diff
@@ -58,28 +58,28 @@ Storage verificaton can only verify the data of l2. When the function result nee
     }
 ```
 
+## Deploying (Sepolia)
 
+Before deploying l1 contracts, deploy l2 contracts on https://github.com/ensdomains/ens-contracts
```
Comment (Arachnid) [inline crosschain-resolver/README.md:63]: Can you be more specific?

File: crosschain-resolver/package.json [hunk 3]
```diff
@@ -7,9 +7,9 @@
     "test": "node scripts/test.js",
     "storage": "hardhat check",
     "clean": "rm -fr artifacts cache node_modules typechain-types",
-    "setupl1": "hardhat run scripts/setupl1.ts --network goerli",
```
Comment (Arachnid) [inline crosschain-resolver/package.json:10]: Perhaps you should have separate entries for different networks?

File: crosschain-reverse-resolver/README.md [hunk 4]
```diff
@@ -7,6 +7,18 @@ For a detailed readme and usage instructions, see the [monorepo readme](https://
 
 ## Deploying (Goerli)
```
Comment (Arachnid) [inline crosschain-reverse-resolver/README.md:8]: Did you mean to delete this header?

**Review by Arachnid — CHANGES_REQUESTED** [review]

File: arb-gateway/README.md (hunk 1 above)
Comment (makoto) [reply arb-gateway/README.md:30]: Yes. it uses the env variable specified on wrangler.toml and that's how to pass the environment

https://github.com/ensdomains/evmgateway/pull/26/files#diff-17f03ef4a17652e904f64bff4ddeb0e9a95bbcf3d0f6f603f3befd4b5fc6d368R14

File: crosschain-resolver/README.md (hunk 2 above)
Comment (makoto) [reply crosschain-resolver/README.md:63]: What needs to be more specific? The exact command to deploy is specified at line 66-68

**Review by Arachnid — CHANGES_REQUESTED** [review]
There's a lot of duplicated code between the op-gateway and the arb-gateway. Can this be refactored into a common package?

**Pull request merged**